### ISSUE TITLE
Remove?

### ISSUE THREAD

**Issue by cjfields** [OBF/Plumbing#3]
Seems like this is unused, so maybe remove until we need it?

**Comment by hlapp** [comment]
I guess so. Can we recreate a repo later after it was deleted earlier? I think so, but am not certain.

**Comment by cjfields** [comment]
@hlapp yep this can be regenerated. I think we set this up with the intent to house scripts and other code used for site maintenance, etc.  This may not be needed though.
